tests/generators/kzg_7594/main.py

```diff
@@ -0,0 +1,518 @@
+"""
+KZG test vectors generator for EIP-7594
+"""
+
+from hashlib import sha256
+from typing import Tuple, Iterable, Any, Callable, Dict
+
+from eth_utils import encode_hex
+
+from eth2spec.utils import bls
+from eth2spec.test.helpers.constants import EIP7594
+from eth2spec.test.helpers.typing import SpecForkName
+from eth2spec.gen_helpers.gen_base import gen_runner, gen_typing
+from eth2spec.eip7594 import spec
+
+
+###############################################################################
+# Helper functions
+###############################################################################
```

Comment:
probably good to move the 4844 duplicate helpers / constants to a `tests/core/pyspec/eth2spec/test/utils/kzg_tests.py` and import everything from it.